### ISSUE TITLE
Tuning set for 2.6

### ISSUE THREAD

**Issue by alexandraBara** [ROCmSoftwarePlatform/MIOpen#270]
Needs tuning:
https://github.com/ROCmSoftwarePlatform/MIOpen/pull/175

Existing entries for `PerformanceImplicitGemmBwdDataV4R1Xdlops` are obsoleted by #232, so all the applicable configs would need retuning.

Add entries from Tuna issue [111](https://github.com/ROCmSoftwarePlatform/Tuna/issues/111)

**use rocm-version 3.6** to build the docker.

**Tagged configs for tuning:**

- Resnet50_v1.5_Pytorch_FP16_256       
- Resnet50_v1.5_Pytorch_FP16_512       
- Resnext101_32x4d_Pytorch_FP16_128    
- Resnext101_32x4d_Pytorch_FP16_256
- nvidia_resnet50v15_bs256_fp16        
- nvidia_resnet50v15_bs256_fp16_fusion 
- nvidia_resnet50v15_bs256_fp32        
- nvidia_resnet50v15_bs256_fp32_fusion 
- recurrent 

**Solvers that need tuning:**

- ImplicitGemmBwdDataV4R1Xdlops (specific arch/num_cu)
- WinogradRxSf2x3 (all arch/num_cu)

**Comment by daniellowell** [comment]
Winograd changes were made as well.

**Comment by sunway513** [comment]
I've collected the following set of configs, please refer to the following confluence:
http://confluence.amd.com/display/MLSE/Convolution+Models+Performance+Optimization#ConvolutionModelsPerformanceOptimization-Resnet50_v1.5,Pytorch,FP16BS256
Please also check the following sections:

- Resnet50_v1.5, Pytorch, FP16 BS 256
- Resnet50_v1.5, Pytorch, FP16 BS 512
- Resnext101_32x4d, Pytorch, FP16 BS 128 
- Resnext101_32x4d, Pytorch, FP16 BS 256

**Comment by JehandadKhan** [comment]
PR #243  would result in changes as well .

**Comment by daniellowell** [comment]
Models | Iterations | Base Batch Size | num_gpu | Precision | 3.5-30 | Diff % | 3.6-22
-- | -- | -- | -- | -- | -- | -- | --
resnet50 | 10 | 256 | 1 | FP16 | 554.58 | -0.83% | 549.99
resnet101 | 10 | 128 | 1 | FP16 | 341.56 | -0.74% | 339.04
resnet152 | 10 | 128 | 1 | FP16 | 238.65 | -0.69% | 237.00
alexnet | 10 | 1024 | 1 | FP16 | 4273.89 | -11.32% | 3790.09
SqueezeNet | 10 | 128 | 1 | FP16 | 1609.62 | 3.30% | 1662.75
inception_v3 | 10 | 256 | 1 | FP16 | 309.88 | -5.46% | 292.97
densenet121 | 10 | 128 | 1 | FP16 | 475.05 | 0.60% | 477.92
vgg16 | 10 | 128 | 1 | FP16 | 330.41 | -16.88% | 274.63
vgg19 | 10 | 128 | 1 | FP16 | 264.94 | -15.71% | 223.31
Resnext101 | 10 | 32 | 1 | FP16 | 144.44 | 22.06% | 176.31
shufflenet_v2_x15 | 10 | 32 | 1 | FP16 | 1020.84 | -10.70% | 911.58


[Tracking ticket for additional tuning.](http://ontrack-internal.amd.com/browse/SWDEV-242236?focusedCommentId=6107673&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-6107673)

**Comment by ekuznetsov139** [comment]
The database currently included with the develop build lacks good MI100 

tunes for resnet50_v1.5 both fp16 and fp32, particularly for conv -F 4 modes. Manual retuning bumps the training rate from 360 image/s to 480 image/s with fp32 and from 500 to 750 with fp16 (TF).
Attaching the configs.
[resnet50.zip](https://github.com/ROCmSoftwarePlatform/MIOpen/files/4881496/resnet50.zip)

**Comment by ekuznetsov139** [comment]
[resnet50.zip](https://github.com/ROCmSoftwarePlatform/MIOpen/files/4881512/resnet50.zip)
